code/modules/mob/living/carbon/human/human.dm

```diff
@@ -502,12 +502,13 @@
 /mob/living/carbon/human/proc/get_idcard(check_hands = FALSE)
 	var/obj/item/card/id/id = wear_id
 	var/obj/item/pda/pda = wear_id
-	if(istype(pda) && pda.id)
-		id = pda.id
-	else
-		pda = wear_pda
+	if(!istype(id)) //We only check for PDAs if there isn't an ID in the ID slot. IDs take priority.
 		if(istype(pda) && pda.id)
 			id = pda.id
+		else
+			pda = wear_pda
+			if(istype(pda) && pda.id)
+				id = pda.id
```

Comment:
This logic is needlessly convoluted. Can you re-write it while you're here? I'm thinking something like this:
```dm
/mob/living/carbon/human/proc/get_idcard(check_hands = FALSE)
	var/obj/item/card/id/id = wear_id
	
	if(!istype(id) && istype(wear_pda) && wear_pda.id) //We only check for PDAs if there isn't an ID in the ID slot. IDs take priority.
		id = wear_pda.id
		
	...
```
If the ID is not an ID, and the thing in our PDA slot is a PDA (covers it being null as well), and it has an ID in it 
Then...
Assign `wear_pda.id` to `id`